### ISSUE TITLE
An alternative way to write modes

### ISSUE THREAD

**Issue by suhr** [SeanArchibald/scale-workshop#37]
Writing modes as a list of intervals between steps is tedious and error prone. It would be nice to write modes as intervals from the pitch center. For example: `0 5 10 15 18 23 25`

**Comment by SeanArchibald** [comment]
Your suggestion would be easy enough to do.

I would prefer to implement a new, user-friendly solution than tweak the existing behaviour (which is based on Scala's approach). What do you think?

For example, the Mode dialog could show a list of all intervals, with a checkbox next to each interval. Checked intervals will be kept in the mode:

- [ ]  9/8
- [ ]  5/4
- [ ]  4/3
- [ ]  3/2
- [ ]  5/3
- [ ]  15/8
- [ ]  2/1

Thanks for the feedback

**Comment by SeanArchibald** [comment]
Or perhaps a hybrid approach where you can either enter numbers (`0 5 10 15 18 23 25`) or tick checkboxes.

**Comment by suhr** [comment]
The hybrid approach is fine.

**Comment by SeanArchibald** [comment]
I have added the alternative way to write modes as you requested.

<img width="312" alt="screenshot 2019-02-24 at 22 15 14" src="https://user-images.githubusercontent.com/11957541/53306198-00183080-3882-11e9-8345-89589ac9a0ae.png">

This addition is in the develop branch currently (http://sevish.com/scaleworkshop-dev/) but will be added to the main Scale Workshop soon. Let me know if you catch any issues with it.

I decided to leave out the checkboxes for now as it hasn't been requested by anyone.
